app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,19 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotation: React.PropTypes.object.isRequired
```

Comment:
I did add it to `choice.cjsx`, but not to `index.cjsx` https://github.com/blog/2265-dismissing-reviews-on-pull-requests. That's were I found it confusing, but it's not a biggie. I pushed latest changes.